app/code/Magento/Sitemap/Model/Sitemap.php

```diff
@@ -192,30 +187,36 @@ class Sitemap extends \Magento\Framework\Model\AbstractModel implements \Magento
     /**
      * Initialize dependencies.
      *
-     * @param \Magento\Framework\Model\Context $context
-     * @param \Magento\Framework\Registry $registry
-     * @param \Magento\Framework\Escaper $escaper
-     * @param \Magento\Sitemap\Helper\Data $sitemapData
-     * @param \Magento\Framework\Filesystem $filesystem
-     * @param ResourceModel\Catalog\CategoryFactory $categoryFactory
-     * @param ResourceModel\Catalog\ProductFactory $productFactory
-     * @param ResourceModel\Cms\PageFactory $cmsFactory
-     * @param \Magento\Framework\Stdlib\DateTime\DateTime $modelDate
-     * @param \Magento\Store\Model\StoreManagerInterface $storeManager
-     * @param \Magento\Framework\App\RequestInterface $request
-     * @param \Magento\Framework\Stdlib\DateTime $dateTime
-     * @param \Magento\Framework\Model\ResourceModel\AbstractResource|null $resource
-     * @param \Magento\Framework\Data\Collection\AbstractDb|null $resourceCollection
-     * @param array $data
-     * @param DocumentRoot|null $documentRoot
-     * @param ItemProviderInterface|null $itemProvider
-     * @param SitemapConfigReaderInterface|null $configReader
+     * @param \Magento\Framework\Model\Context                        $context
```

Comment:
Please, revert changes for doc block, Magento static tests will be failed due to specific coding standards

Reply:
Doc block has been changed because class inheritance has been changed.